hokusai/staging.yml

```diff
@@ -21,28 +20,64 @@ spec:
     spec:
       containers:
         - name: aprb-web
+          env:
+            - name: PORT
+              value: "4000"
           envFrom:
-          - configMapRef:
-              name: aprb-environment
+            - configMapRef:
+                name: aprb-environment
           image: 585031190124.dkr.ecr.us-east-1.amazonaws.com/aprb:staging
           imagePullPolicy: Always
           ports:
-          - containerPort: 4000
+            - name: aprb-http
+              containerPort: 4000
+          readinessProbe:
+            httpGet:
+              port: aprb-http
+              path: /ping
+              httpHeaders:
+                - name: X-FORWARDED-PROTO
+                  value: https
+            initialDelaySeconds: 5
+            periodSeconds: 15
+            timeoutSeconds: 10
+          resources:
+            requests:
+              cpu: 700m
+              memory: 1024Mi
+            limits:
+              memory: 1536Mi
       dnsPolicy: ClusterFirst
       dnsConfig:
         options:
-        - name: ndots
-          value: "1"
+          - name: ndots
+            value: "1"
       affinity:
         nodeAffinity:
           preferredDuringSchedulingIgnoredDuringExecution:
-          - weight: 1
-            preference:
-              matchExpressions:
-              - key: tier
-                operator: In
-                values:
-                - foreground
+            - weight: 1
+              preference:
+                matchExpressions:
+                  - key: tier
+                    operator: In
+                    values:
+                      - foreground
+
+---
+apiVersion: autoscaling/v1
+kind: HorizontalPodAutoscaler
+metadata:
+  name: aprb-web
+  namespace: default
+spec:
+  scaleTargetRef:
+    apiVersion: extensions/v1beta1
+    kind: Deployment
+    name: aprb-web
+  minReplicas: 2
```

Comment:
i think this can be 1 and max replicas maybe 2?

Reply:
We've been using a minimum of 2 and then `minReplicas` x3 as the max for these, but in staging could probably get by with 3 as the max.